letsencrypt_transip_nl/models/base_config_settings.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+from tempfile import NamedTemporaryFile
+import base64
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from transip.service.objects import DnsEntry
+    from transip.service import DomainService
+except ImportError:
+    _logger.error('`transip` Python library is missing.')
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    letsencrypt_dns_provider = fields.Selection(
+        selection_add=[('transip', 'TransIP')],
+    )
+
+    def letsencrypt_transip_support(self, challenge, domain):
```

Comment:
I still don't understand why this is a function of the config wizard and not of the letsencrypt abstract model